packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -180,9 +180,18 @@ export class Button extends FormAssociatedButton {
     };
 
     /**
-     * References the root element
+     * This has been deprecated in favor of control property
+     * @deprecated
```

Comment:
```suggestion
     * References the root element
     * @deprecated This has been deprecated in favor of control property
```

Going to try and commit this since it still shows failure.